go/tasks/plugins/array/k8s/transformer.go

```diff
@@ -29,6 +32,22 @@ func (a *arrayTaskContext) InputReader() io.InputReader {
 	return a.arrayInputReader
 }
 
+func GetNamespaceForExecution(tCtx core.TaskExecutionContext) string {
```

Comment:
Can you make it take `namespaceTemplate` (or the entire config if needed)? I try to avoid calling GetConfig() outside the initialization code to make it easier to unit test these functions...

```suggestion
func GetNamespaceForExecution(tCtx core.TaskExecutionContext, namespaceTemplate string) string {
```

Reply:
done